Sources/Commands/SwiftTestCommand.swift

```diff
@@ -132,6 +148,20 @@ struct SharedOptions: ParsableArguments {
         // Default to disabled since swift-testing is experimental (opt-in.)
         return false
     }
+
+    /// Get the set of enabled testing libraries.
+    func enabledTestingLibraries(swiftCommandState: SwiftCommandState) throws -> Set<BuildParameters.Testing.Library> {
```

Comment:
Formatting nit: we avoid long lines for better readability in diff views and smaller screen sizes.
```suggestion
    func enabledTestingLibraries(
        swiftCommandState: SwiftCommandState
    ) throws -> Set<BuildParameters.Testing.Library> {
```

Reply:
What's the cutoff you generally prefer? I can set my gutter to that.

Reply:
We have all of the formatting rules specified in the formatter config https://github.com/apple/swift-package-manager/blob/main/.swiftformat. Just run `./Utilities/soundness.sh` before committing and that will reformat changed files for you with that config.

Reply:
That script did not produce any diagnostics nor make any changes to my branch.